### PR TITLE
Icon & attachment attribute aware

### PR DESCRIPTION
**Pull request by apepper**
Needs content: https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://icon-attachment-attribute-aw.scrivito-portal-app.pages.dev/en/my-tynacoon?_scrivito_workspace_id=jf6431200f480927&_scrivito_display_mode=view

### CODE REVIEW

File: src/Widgets/index.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { initNeoletterFormWidgets } from 'scrivito-neoletter-form-widgets'
 import.meta.glob(['./**/*WidgetClass.ts', './**/*WidgetComponent.tsx'], {
   eager: true,
 })
+import './migrateAttributeNameToData'
```
Comment (dcsaszar) [inline src/Widgets/index.ts]: * [ ] Why make this part of the code history? It looks like a one-shot migration that can be run on the console.

File: src/Widgets/index.ts [hunk 2]
```diff
@@ -2,7 +2,6 @@ import { initNeoletterFormWidgets } from 'scrivito-neoletter-form-widgets'
 import.meta.glob(['./**/*WidgetClass.ts', './**/*WidgetComponent.tsx'], {
   eager: true,
 })
-import './migrateAttributeNameToData'
```
Comment (dcsaszar) [inline src/Widgets/index.ts]: * [ ] 😿 Drop the last two commits from the history?

**Review by dcsaszar — CHANGES_REQUESTED** [review]
LGTM. I only briefly checked the content changes.

See comments for a potentially cleaner history.

File: src/Widgets/index.ts (hunk 1 above)
Comment (apepper) [reply src/Widgets/index.ts]: It is a one-shot migration. But not a trivial one, that is why I wanted to have it at least in a file, that I can later restore if needed.

Comment (dcsaszar) [reply src/Widgets/index.ts]: How about a branch in a fork?
Making the script part of the history has some drawbacks:
* It creates noise in the history
* There are no guidelines for a reviewer for how to review code and/or functionality

Comment (apepper) [reply src/Widgets/index.ts]: I forced pushed a version without those two commits.

**Comment by apepper** [comment]
REVIEW ([b771287](https://github.com/Scrivito/scrivito-portal-app/pull/439/commits/b77128778e50be836f3857ffb18ad98d67d7ece2))

**Review by dcsaszar — APPROVED** [review]
🏄

**Pull request merged**